index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+      <div id="img-div">
+        <img
+          src="/images/tribute-page-main-image.jpeg"
+          alt="tribute image"
+          id="image"
+        />
```

Comment:
The image source path is broken and the tribute-page-main.jpeg is not displaying when opened locally.
Please correct the path.